content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,215 @@
+2. In the top right corner, click the **New Test** button. 
+3. Select **API test**.
+4. Define the configuration of your API test.
+
+{{< tabs >}}
+
+{{% tab "HTTP Test" %}}
+
+{{< img src="synthetics/api_tests/make-http-request.png" alt="Make HTTP Request" responsive="true" style="width:80%;" >}}
+
+Define the request you want to be executed by Datadog:
+
+1. **Choose request type**: `HTTP`
+2. Choose the **Method** and **URL** to query. Available methods are: `GET`, `POST`, `PATCH`, `PUT`, `HEAD`, `DELETE`, and `OPTIONS`.
+3. **Name**: Set the name of your API test.
+4. **Select your tags**: Select the tags attached to your API test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the [Synthetics page][1].
+5. **Locations**: Select which Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetics API test on a private endpoint not accessible from the public internet.
+6. **How often should Datadog run the test?** Set the interval for how often the test will run. Intervals are available between every one minute to once per week.
+7. Click on **Test URL** to try out the request configuration. You should see a response preview show up on the right side of your screen.
+
+[1]: /synthetics
+[2]: /api/?lang=bash#get-available-locations
+[3]: /synthetics/private_locations
+{{% /tab %}}
+
+{{% tab "SSL Test" %}}
+
+{{< img src="synthetics/api_tests/make-ssl-request.png" alt="Make SSL Request" responsive="true" style="width:80%;" >}}
+
+1. **Choose request type**: `SSL`
+2. Specify the `Host` and the SSL `Port`. By default, the port is set to _443_.
+3. **Name**: The name of your API test.
+4. **Select your tags**: Select the tags attached to your API test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the [Synthetics page][1].
+5. **Locations**: Select which Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetics API test on a private endpoint not accessible from the public internet.
+6. **How often should Datadog run the test?** Set the interval for how often the test will run. Intervals are available between every five minutes to once per week.
+7. Click on **Test Connection** to try out the request configuration. You should see a response preview show up on the right side of your screen.
+[1]: /synthetics
+[2]: /api/?lang=bash#get-available-locations
+[3]: /synthetics/private_locations
+{{% /tab %}}
+
+{{< /tabs >}}
+
+#### Set alert conditions and assertions
+
+Once the URL is tested, a response preview will populate in a panel to the right of the configuration options. If the test fails, review the error [here][6].
+
```

Comment:
I feel like after this section's first sentence, we should clarify the difference between assertions & alert conditions. I'm thinking of sthg like this:

First: assertions

> This panel shows the request as well as the response preview of your API call. It also automates the creation of assertions on the status code, `content-type` header, and response time of the request.

> You can create custom assertions: in the **Headers** section of this panel's response preview, click on any header to auto-populate a `header` assertion. It is also possible to manually add [assertions][8], on `body`, `header`, `response type`, or `status code` by clicking on **New Assertion**.

> For example, set an assertion on status code being a 200, and select the server listed in the response preview header. This will send you a notification when there is a server error on the server that's assimilated with your API endpoint.

Second: alert conditions

> Alert conditions determine the circumstances under which you want a test to send a notification alert. 

> You can decide to be alerted after a specific amount of downtime and/or when failures are happening on a specific number of locations. To configure, set the pre-populated input areas in the given alert condition:

> `An alert is triggered if your test fails for x minutes from any x of x locations. `

> By default, Synthetics tests do not retry after a failed result for a given location. If you want a location to be considered failed only after a certain number of retries, pre-populate the given alert condition:

> `Retry x times before location is marked as failed.`

> Configure the minutes, locations, and retry time and an alert will automatically generate when these conditions are met.

Only then should we start mentioning the message that can be configured, etc.